### PR TITLE
Add new hexstickers

### PR DESCRIPTION
**Pull request by jchojna**
- shiny.react
- shiny.fluent
- data.validator

### CODE REVIEW

File: app/scripts/libraries.js [hunk 1]
```diff
@@ -58,5 +58,35 @@ export const libraries = [
...
+      'This R package enables using React in Shiny apps and is used e.g. by the shiny.fluent package. It contains R and JS code which is independent from the React library (e.g. Fluent UI) that is being wrapped.'
+    ],
+    'repoLink': 'https://github.com/Appsilon/shiny.react',
+    'demoLink': 'https://demo.appsilon.com/apps/fluentui/'
+  },
+  {
+    'id': 'shiny-fluent',
+    'heading': 'shiny.fluent',
+    'paragraphs': [
+      'We believe that a great UI plays a huge role in the success of application projects. shiny.fluent gives your apps beautiful and professional look, rich set of components easily usable in Shiny and fast speed of development that Shiny is famous for.',
```
Comment (MicahAppsilon) [inline app/scripts/libraries.js:76]:
```suggestion
      'We believe that a great UI plays a huge role in the success of application projects. shiny.fluent gives your apps a beautiful and professional look, rich set of easy-to-use components in Shiny, and fast speed of development that Shiny is famous for.',
```

**Review by przytu1 — CHANGES_REQUESTED** [review]
Code looks good! @jchojna please address @MicahAppsilon comments, push the update, and then you can merge :heavy_check_mark:

File: app/scripts/libraries.js [hunk 2]
```diff
@@ -58,5 +58,35 @@ export const libraries = [
     ],
     'repoLink': 'https://github.com/Appsilon/shiny.i18n',
     'demoLink': 'https://appsilon.github.io/shiny.i18n/'
+  },
+  {
+    'id': 'shiny-react',
+    'heading': 'shiny.react',
+    'paragraphs': [
+      '',
+      'This R package enables using React in Shiny apps and is used e.g. by the shiny.fluent package. It contains R and JS code which is independent from the React library (e.g. Fluent UI) that is being wrapped.'
```
[the remark this answers is not in the document]
Comment (jchojna) [reply app/scripts/libraries.js]: fixes

File: app/scripts/libraries.js (hunk 1 above)
Comment (jchojna) [reply app/scripts/libraries.js]: fixed

**Pull request merged**